lib/static/components/state/screenshot/with-synced-scale.jsx

```diff
@@ -1,116 +1,116 @@
-import React, {Component} from 'react';
+import useResizeObserver from '@react-hook/resize-observer';
```

Comment:
Got rid of `rc-resize-observer` in favor of more modern hook-based solution.

rc-resize-observer uses obsolete API and it's unclear when it will be fixed: https://github.com/react-component/resize-observer/issues/208